src/mono/netcore/nuget/Microsoft.NET.Runtime.Android.Sample.Mono/Microsoft.NET.Runtime.Android.Sample.Mono.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
   <ItemGroup>
-    <ProjectReference Include="$(RepoToolsLocalDir)tasks\mobile.tasks\AndroidAppBuilder\AndroidAppBuilder.csproj" />
+    <ProjectReference Include="$(RepoTasksDir)mobile.tasks\AndroidAppBuilder\AndroidAppBuilder.csproj" />
```

Comment:
Do we need the `mobile.tasks` intermediate directory? Would it make more sense to have the tasks directly in the `RepoTasksDir` ?

Reply:
@akoeplinger @steveisok @marek-safar are more suitable to answer this :)

Reply:
remove the intermediate directory sounds fine to me.

Reply:
done